definitions/infra-powerdns_recursor/summary_metrics.yml

```diff
@@ -0,0 +1,23 @@
+concurrent_queries:
+  title: Number of concurrent queries.
+  unit: COUNT
+  query:
+    select: average(powerdns_recursor_concurrent_queries)
+    from: Metric
+  eventId: entity.guid
```

Comment:
These event id's are part of the `query` so they should go one tab in. [See docs](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/summary_metrics.md#defining-summary-metrics)